src/app/stores/chart.ts

```diff
@@ -286,6 +290,23 @@ export class ChartStore extends BaseStore {
 
       this.solveConstraintsAndUpdateGraphics();
     }
+    if (action instanceof Actions.AddGlyph) {
+      this.parent.saveHistory();
+      const glyph = this.chartManager.addGlyph(
+        action.classID,
+        this.datasetStore.dataset.tables[0].name
```

Comment:
Not that you need to change this or rewrite everything.  It just seems a little weird, conceptually, that a glyph is bound to a specific table.